### PR TITLE
Feature/submission form confirmation

### PR DESCRIPTION
**Pull request by mr-ryan12**
## Checklist
- [ ] Tests written where required
- [ ] All tests passing
- [ ] Unused code/prys/etc deleted
Testing Coverage at <percentage>%

## Changes Made
  
What (if any) features are you implementing?
- Added a feature that displays a modal for a new submission
- Uses an animation to indicate the submission form is uploaded
- Allows the users to go 'Home' or view their submission on completion

What (if anything) did you refactor?
- Refactored 'SubmissionForm.js'

Were there any issues that arose?
- Not at the time

Is there anything that you need from your teammate?
- PR review, comments and merge

Any other comments, questions, or concerns?
- Need to still test for opening and closing the confirmation modal, failed fetches (w/ error handling) and navigation links

### CODE REVIEW

File: src/components/SubmissionForm/SubmissionForm.js [hunk 1]
```diff
@@ -44,14 +52,22 @@ const SubmissionForm = () => {
...
+      setIsNotImageFile(true)
     } else {
       setIsLargeFile(false)
+      setIsNotImageFile(false)
       setProfileImage(event.target.files[0])
     }
   }
 
+  useEffect(() => {
+    return () => {setIsUploading(false)}
```
Comment (aliroemhildt) [inline src/components/SubmissionForm/SubmissionForm.js:65]: Not sure if having this return a callback is necessary or if we can just call setIsUploading(false) inside of the useEffect hook. I think it's fine to leave for now if it's working - but is something we can look into when refactoring!

File: src/components/SubmissionForm/SubmissionForm.js [hunk 2]
```diff
@@ -71,7 +87,11 @@ const SubmissionForm = () => {
       body: formData
     })
     .then(response => response.json())
-    .then(data => console.log(data))
+    .then(data => {
+      console.log(data)
+      setSubmissionId(data.data.id)
+      setIsResolved(true)
```
Comment (aliroemhildt) [inline src/components/SubmissionForm/SubmissionForm.js:93]: super smart way to handle this 👍

File: src/components/UploadingModal/UploadingModal.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import UploadingSpinner from '../UploadingSpinner/UploadingSpinner'
+import './UploadingModal.scss'
+
+const UploadingModal = props => {
```
Comment (aliroemhildt) [inline src/components/UploadingModal/UploadingModal.js:6]: we should destructure props here to be consistent with other components - something to hit when we go through and refactor.

**Review by oacortez — APPROVED** [review]
Thank you for creating this branch, Ryan! I really like that you implemented this feature it looks spotless and as well I really like that you added a nice touch of animation to let the user know that their submission form is getting sent!

**Review by aliroemhildt — APPROVED** [review]
Great work Ryan - this loading modal is awesome!

**Pull request merged**